### PR TITLE
Added a GeometryStream test

### PR DESCRIPTION
**Pull request by saeeedtorabi**
imodel-native: https://github.com/iTwin/imodel-native/pull/775

### CODE REVIEW

**Comment by pmconne** [comment]
Change target branch back to master, imodel02 is no longer used.

File: core/backend/src/test/standalone/GeometryStream.test.ts [hunk 1]
```diff
@@ -1586,6 +1586,43 @@ describe("ElementGeometry", () => {
     assert(IModelStatus.Success === doElementGeometryValidate(imodel, newId, expected, false, elementProps));
   });
 
+  it.only("create GeometricElement3d from local coordinate interpolation curve flatbuffer data", async () => {
```
Comment (pmconne) [inline core/backend/src/test/standalone/GeometryStream.test.ts:1589]: rm .only

Comment (pmconne) [inline core/backend/src/test/standalone/GeometryStream.test.ts:1589]:
```suggestion
  it("create GeometricElement3d from local coordinate interpolation curve flatbuffer data", async () => {
```

Comment (saeeedtorabi) [reply core/backend/src/test/standalone/GeometryStream.test.ts]: Done

**Review by dassaf4 — APPROVED** [review]

**Review by pmconne — APPROVED** [review]

**Comment by pmconne** [comment]
Please monitor [this release pipeline](https://dev.azure.com/bentleycs/iModelTechnologies/_build/results?buildId=2927584&view=results). If all goes well, it will automatically merge a new addon into this PR. If all doesn't go well, notify me.

**Review by pmconne — APPROVED** [review]

**Pull request merged**